### PR TITLE
Update postcss-plugins.md

### PR DESCRIPTION
**Pull request by core01**
If there will be postcss.config.js, nuxt.config.js postcss property will be ignored

Reproduce link: https://github.com/core01/postcss-simple-vars-nuxt-reproduction

### CODE REVIEW

**Comment by core01** [comment]
@farzadso done, check it please!

**Comment by farzadso** [comment]
Thanks @core01 

It looks much better now.

I would also suggest adding back-ticks before and after `postcss.config.js`

**Comment by core01** [comment]
@farzadso back-ticks added

File: en/faq/postcss-plugins.md [hunk 1]
```diff
@@ -5,7 +5,20 @@ description: How to add PostCSS plugins?
 
 # How to add PostCSS plugins?
 
-In your `nuxt.config.js` file:
+### Default Method
```
Comment (manniL) [inline en/faq/postcss-plugins.md]: I'd probably switch the two heading. The "default" should be using `nuxt.config.js`. The "legacy" or "traditional" way is a postcss.config.js

Comment (core01) [reply en/faq/postcss-plugins.md]: @manniL will it good if I also swap two methods? Then the first will be default (nuxt.config.js) and the second (postcss.config.js) ?

Comment (manniL) [reply en/faq/postcss-plugins.md]: Yes, please. :relaxed:

File: en/faq/postcss-plugins.md [hunk 2]
```diff
@@ -5,7 +5,8 @@ description: How to add PostCSS plugins?
...
```
Comment (manniL) [inline en/faq/postcss-plugins.md]: Maybe
```suggestion
### Recommended Method
```

File: en/faq/postcss-plugins.md [hunk 3]
```diff
@@ -5,7 +5,8 @@ description: How to add PostCSS plugins?
...
+Delete the `postcss.config.js` in your project directory (if existent). Then, in your `nuxt.config.js` file add the following:
```
Comment (manniL) [inline en/faq/postcss-plugins.md]:
```suggestion
If present, rename or delete the `postcss.config.js` in your project directory. Then, in your `nuxt.config.js` file add the following:
```

Comment (core01) [reply en/faq/postcss-plugins.md]: @farzadso what do you think about it?

Comment (farzadso) [reply en/faq/postcss-plugins.md]: @core01 Couldn't agree any more. It's got much better sentencing.

**Comment by manniL** [comment]
Thanks ☺️

**Pull request merged**